woocommerce/woocommerce-ios
# [Mobile Payments]Remove the upsell card readers banner from the payment methods screen.

## Description

With this PR we remove the upsell card readers banner from the Payment Method screen. The reasons are a few: it's been there for a while already, it doesn't bring much impact on this screen, we are promoting Tap to Pay with JITMs at the moment, and Android already removed it. More context here pdqJU4-2t6-p2#comment-873
As that campaign is not used anymore, we remove any references across the app.

## Testing instructions


From a clean install:

1. Go to Menu
2. Go to Payments
3. Tap on Collect Payments
4. Follow the Simple Payments flow
5. Ensure that no Upsell Card Readers banner is shown on the Payment Methods Screen (See screenshot)

## Screenshots

<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/626e998e-ec39-44e5-8ba2-4d3312c0fc27" width="375">

---
